girder-dandi-archive/girder_dandi_archive/rest.py

```diff
@@ -70,10 +70,19 @@ def create_dandiset(self, params):
         )["value"]
         padded_identifier = f"{new_identifier_count:0{DANDISET_IDENTIFIER_LENGTH}d}"
 
+        user = self.getCurrentUser()
         meta = {
             "name": name,
             "description": description,
             "identifier": padded_identifier,
+            "contributor": [
+                {
+                    "identifier": "",
+                    "name": user["firstName"] + " " + user["lastName"],
```

Comment:
I'd prefer using f-strings here:
```suggestion
                    "name": f"{user['firstName']} {user['lastName']}",
```